.circleci/config.yml

```diff
@@ -159,3 +159,334 @@ jobs:
+            docker exec gcs-emulator netstat -tuln || echo "netstat not available"
+
+            echo "Trying port 4443..."
+            docker run --rm --network host appropriate/curl -s -v "http://localhost:4443/storage/v1/b" || echo "Port 4443 failed"
+            
+            # Create the bucket using Docker network with correct port 4443
+            docker run --rm --network host appropriate/curl -X POST \
+              "http://localhost:4443/storage/v1/b?project=test-project" \
+              -H "Content-Type: application/json" \
+              -d '{"name": "test-bucket"}'
+            
+            # Verify bucket exists
+            echo "Verifying bucket exists..."
+            docker run --rm --network host appropriate/curl -s \
+              "http://localhost:4443/storage/v1/b?project=test-project"
+      - run:
+          name: Setup test data
+          command: |
+            source ~/miniconda/etc/profile.d/conda.sh
+            conda activate expertise
+            cd ~/openreview-expertise
+            python -m pytest tests/test_create_conference.py tests/test_create_dataset.py
+      
+      # 5. Build the container
+      - run:
+          name: Build container
+          command: |
+            docker build -t expertise-test:latest .
+      # 7. Verify CUDA exists
+      - run:
+          name: Verify CUDA
+          command: |
+            # Check if the expected files exist and have correct format
+            source ~/miniconda/etc/profile.d/conda.sh
+            conda activate expertise
+            python ~/openreview-expertise/tests/container_tests/verify_cuda_support.py
+      
+      # 8. Run group-paper pipeline
+      - run:
+          name: Generate Token for Group-Paper Pipeline
+          command: |
+            source ~/miniconda/etc/profile.d/conda.sh
+            conda activate expertise
+            # Print working directory for debugging
+            pwd
+            # Execute the token generation script
+            python ~/openreview-expertise/tests/container_tests/generate_token.py ~/openreview-expertise/tests/container_jsons/container_paper_group.json
+            # Verify file exists
+            ls -la test_input.json
+      - run:
+          name: Run Group-Paper Pipeline
+          command: |
+            # Capture container output to a log file
+            echo "Running expertise container with output captured to log..."
+            mkdir -p logs
+            
+            # Run with tee to capture output while still displaying it
+            docker run --network host \
+              -e GOOGLE_CLOUD_PROJECT=test-project \
+              -e STORAGE_EMULATOR_HOST=http://localhost:4443 \
+              -e PYTHONUNBUFFERED=1 \
+              -e DEBUG=1 \
+              -e LOG_LEVEL=DEBUG \
+              -v $(pwd)/test_output:/app/output \
+              --entrypoint="" \
+              expertise-test:latest \
+              python -m expertise.execute_pipeline "$(cat test_input.json)"
```

Comment:
Is this a test? Does it mean we can't test this using pytest?

Reply:
Yes this is a test - we do have corresponding tests in the `test_pipeline.py` file, but the point of these tests in CircleCI is to make sure that they can run properly in the image - it's as close as we can get to testing the Vertex AI Pipeline